index.js

```diff
@@ -0,0 +1,11 @@
+"use strict"
+
+const App = () => {
+    return (
+        <div>Hello world</div>
+    );
+}
+
+
+const domContainer = document.querySelector('app');
+ReactDOM.render(e(App), domContainer);
```

Comment:
Remove this file it is not longer needed since we never got to implement this app in react.